joomla/joomla-cms
# New Feature to allow filtering the popular tags module by a Parent Tag.

Pull Request for Issue #17466  .

### Summary of Changes
Added a Parent filter Tag to the Popular Tags Module

### Testing Instructions
Apply Patch, select a parent tag or tags in the Popular Tags Module, results should only include the child tags.
